src/test/java/plugins/CompressArtifactsPluginTest.java

```diff
@@ -93,12 +93,12 @@ public void archiveLargerThan4GInTotal() throws Exception {
         job.addPublisher(ArtifactArchiver.class).includes("*");
         if (SystemUtils.IS_OS_UNIX) {
             job.addBuildStep(ShellBuildStep.class).command( // Generate archive larger than 4G
-                "#!/bin/bash\nwget $JENKINS_URL/jnlpJars/jenkins-cli.jar -O stuff.jar; for i in {0..7000}; do cp -l stuff.jar stuff.${i}.jar; done"
+                "#!/bin/bash\nwget $JENKINS_URL/jnlpJars/jenkins-cli.jar -O stuff.jar; for i in {0..7000}; do cp stuff.jar stuff.${i}.jar; done"
```

Comment:
Not described in the original issue, but this also makes the test fail on Mac OS since the '-l' option is not valid on this OS.